test/e2e/federated-service.go

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
This shouldn't be renamed to "federation-service" like the way we call other items that belong to the federation. This service is "federated" across a federation, a verb, in addition to being part of the federation.